apps/rewards-dashboard/src/layouts/DashboardLayout.tsx

```diff
@@ -67,29 +76,32 @@ const ConnectedDashboard = () => {
     enabled: !!address,
     staleTime: 5000,
     queryFn: () => {
-      return fetch(`${WAC_URL}/usersummary?address=${address}`).then((res) => {
-        return res.json();
-      });
+      return fetch(`${WAC_URL}usersummary.modal.run?address=${address}`).then(
+        (res) => {
+          return res.json();
+        }
+      );
     },
   });
 
   useEffect(() => {
+    console.log(userInfo);
```

Comment:
console.log should be removed

Reply:
Thanks for the feedback. Removed the logs